src/js/stores/AppsStore.js

```diff
@@ -115,6 +116,14 @@ function getAppHealthWeight(health) {
   }, 0);
 }
 
+function getAppType(app) {
```

Comment:
Yeah, generally it always makes sense to do slightly unrelated changes in a separate PR.
Although here I cannot see a dependency on an array, it's all based on these type-strings.
So this could be done in this PR, but you can decide. Do what you prefer.